system-veriables: add @@tidb_min_paging_size and @@tidb_max_paging_size (#9813)
pingcap/docs: system-variables.md

- Range: `[100, 16384]`
- This variable is used to set the maximum number of schema versions (the table IDs modified for corresponding versions) allowed to be cached. The value range is 100 ~ 16384.

> ### tidb_max_paging_size

Reviewer: ```suggestion
### tidb_max_paging_size <span class="version-mark">New in v6.2.0</span>
```

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Type: Integer
- Default value: `50000`
- Range: `[1, 2147483647]`
- Unit: Rows

> - This variable is used to set the maximum number of rows during the coprocessor paging request process. Setting it to a too small value increases the RPC count between TiDB and TiKV, while setting to too large of a value may cause TiDB use more memory in some cases, such as load data or full table scan.

Reviewer: ```suggestion
- This variable is used to set the maximum number of rows during the coprocessor paging request process. Setting it to a too small value increases the RPC count between TiDB and TiKV, while setting it to a too large value results in excessive memory usage in some cases, such as data loading and full table scan.
```

Reviewer 2: ```suggestion
- This variable is used to set the maximum number of rows during the coprocessor paging request process. Setting it to too small a value increases the RPC request count between TiDB and TiKV, while setting it to too large a value might cause TiDB to use more memory in some cases, such as loading data or full table scan.
```

> - This variable is used to set the maximum number of rows during the coprocessor paging request process. Setting it to an excessively small value increases the RPC count between TiDB and TiKV, while setting it to an excessively large value results in excessive memory usage in some cases, such as loading data and full table scan.

Reviewer: ```suggestion
- This variable is used to set the maximum number of rows during the coprocessor paging request process. Setting it to too small a value increases the RPC count between TiDB and TiKV, while setting it to too large a value results in excessive memory usage in some cases, such as loading data and full table scan.
```

- Unit: Seconds
- This variable is used to set the step of the Prometheus statement generated when querying `METRICS_SCHEMA`.

> ### tidb_min_paging_size

Reviewer: ```suggestion
### tidb_min_paging_size <span class="version-mark">New in v6.2.0</span>
```

- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
- Type: Integer
- Default value: `128`
- Range: `[1, 2147483647]`
- Unit: Rows

> - This variable is used to set the minimal number of rows during the coprocessor paging request process. Setting it to a too small value increases the RPC count between TiDB and TiKV, while setting to too large of a value may cause performance decrease when queries using IndexLookup with Limit.

Reviewer: ```suggestion
- This variable is used to set the minimal number of rows during the coprocessor paging request process. Setting it to a too small value reduces the RPC count between TiDB and TiKV, while setting it to a too large value results in performance degrade when queries use IndexLookup with Limit.
```

Reviewer 2: ```suggestion
- This variable is used to set the minimum number of rows during the coprocessor paging request process. Setting it to too small a value increases the RPC request count between TiDB and TiKV, while setting to too large a value might cause performance decrease when executing queries using IndexLookup with Limit.
```

> - This variable is used to set the minimum number of rows during the coprocessor paging request process. Setting it to too small a value increases the RPC request count between TiDB and TiKV, while setting to too large a value might cause performance decrease when executing queries using IndexLookup with Limit.

Reviewer 2: ```suggestion
- This variable is used to set the minimum number of rows during the coprocessor paging request process. Setting it to too small a value increases the RPC request count between TiDB and TiKV, while setting it to too large a value might cause performance decrease when executing queries using IndexLookup with Limit.
```
